tests/test_models.py

```diff
@@ -1,27 +1,87 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 """
 Tests for the `edx-enterprise` models module.
 """
 
 from __future__ import absolute_import, unicode_literals
 
+import datetime
+import unittest
 
-class TestEnterpriseCustomer(object):
+import ddt
+import freezegun
+from pytest import mark
+
+from enterprise.models import EnterpriseCustomer
+from test_utils.factories import (
+    EnterpriseCustomerFactory, EnterpriseCustomerHistoryFactory, EnterpriseCustomerMemberFactory
+)
+
+
+@mark.django_db
+class TestEnterpriseCustomerManager(unittest.TestCase):
     """
-    Tests of the EnterpriseCustomer model.
+    Tests for enterprise customer manager
     """
+    def tearDown(self):
+        super(TestEnterpriseCustomerManager, self).tearDown()
+        EnterpriseCustomer.objects.all().delete()
 
-    def test_something(self):
-        """TODO: Write real test cases."""
-        pass
+    def test_get_queryset_returns_only_active(self):
+        customer1 = EnterpriseCustomerFactory(active=True)
+        customer2 = EnterpriseCustomerFactory(name="Customer 2", active=True)
```

Comment:
The names here don't actually matter. Remove them, and update your factories to generate fuzzy text.